### PR TITLE
Remove flakey amp validator unit test

### PR DESCRIPTION
**Pull request by HarryVerhoef**
**Overall change:**
Removes a flakey time-consuming amp-validator unit test which adds little to no value.

**Code changes:**

- Removes the unit test and a mock.

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [X] I have assigned this PR to the Simorgh project
- [X] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [X] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by ryanmccombe — APPROVED** [review]

File: scripts/ampHtmlValidator/index.test.js [hunk 1]
```diff
@@ -91,6 +93,8 @@ describe('amp validator tests', () => {
   });
 
   it('should print passes when verbose is true', async () => {
+    jest.setTimeout(10000);
```
Comment (andrewscfc) [inline scripts/ampHtmlValidator/index.test.js]: Does this test really have much value, it just checks the validator logs and doesn't even check content of the logs? I wonder if we should improve it or delete it?

File: yarn.lock [hunk 2]
```diff
@@ -1578,22 +1578,6 @@
     enabled "2.0.x"
     kuler "^2.0.0"
 
-"@electron/get@^1.0.1":
```
Comment (andrewscfc) [inline yarn.lock]: I've seen these changes in a few PRs, they relate the electron dependency being removed but the lock file not being updated. The first PR to go in should resolve these

File: scripts/ampHtmlValidator/index.test.js (hunk 1 above)
Comment (HarryVerhoef) [reply scripts/ampHtmlValidator/index.test.js]: Good point @andrewscfc, I think this test could be replaced by tests that check for 0 and non-zero exit codes for pass and fail respectively? Would surely be better than checking the logs

Comment (andrewscfc) [reply scripts/ampHtmlValidator/index.test.js]: Yeah, just having a 10 second test in our suite with little value seems a bit silly

Comment (HarryVerhoef) [reply scripts/ampHtmlValidator/index.test.js]: Agreed, on it 👍

Comment (HarryVerhoef) [reply scripts/ampHtmlValidator/index.test.js]: I added some tests which check exit codes and pass/fail status for valid and invalid amp html, but when running `yarn test:unit` they all timeout, so I I'll just delete the original flakey test.

**Review by andrewscfc — APPROVED** [review]

**Pull request merged**